lib/cypress_on_rails/command_executor.rb

```diff
@@ -13,12 +13,12 @@ def self.perform(file,command_options = nil)
       raise e
     end
 
-    def self.load_cypress_helper
-      cypress_helper_file = "#{configuration.cypress_folder}/cypress_helper"
-      if File.exist?("#{cypress_helper_file}.rb")
-        Kernel.require cypress_helper_file
+    def self.load_e2e_helper
+      e2e_helper_file = "#{configuration.install_folder}/e2e_helper.rb"
+      if File.exist?(e2e_helper_file)
+        Kernel.require e2e_helper_file
       else
-        logger.warn "could not find #{cypress_helper_file}.rb"
+        logger.warn "could not find #{e2e_helper_file}"
```

Comment:
what happened to the fallback to look for `cypress_helper.rb` so we don't break any existing setups?

Reply:
Thank you @grantspeelman . Reverted back.